lib/l10n/app_en.arb

```diff
@@ -135,5 +135,11 @@
     "sensorGPSError": "No GPS Fix",
     "sensorAcceleration": "Acceleration",
     "sensorFinedust": "Finedust",
-    "sensorDistanceShort": "Distance"
+    "sensorDistanceShort": "Distance",
+    "tagLoadError": "Could not load list of locations.",
+    "locationWiesbaden": "Wiesbaden",
+    "locationMuenster": "Münster",
+    "locationArnsberg": "Arnsberg",
+    "locationOther": "Other",
+    "selectLocation": "Select location"
```

Comment:
Instead of `"selectLocation": "Select location"` let's use `"campaignTagLabel": "Campaign"` or similar